docs/components/connectors/out-of-the-box-connectors/graphql.md

```diff
@@ -0,0 +1,212 @@
+---
+id: graphql
+title: GraphQL Connector
+description: Execute a GraphQL query or mutation from your BPMN process.
+---
+
+The **GraphQL Connector** allows you to execute a GraphQL query or mutation from your BPMN process.
+
+## Prerequisites
+
+To use the GraphQL Connector, a GraphQL server must be configured. A respective authentication method needs to be configured. Follow [these steps in the appendix](#appendix) to learn how to set this up.
+
+## Create a GraphQL Connector task
+
+To use a **GraphQL Connector** in your process, either change the type of an existing task by clicking on it and using the wrench-shaped **Change type** context menu, or create a new Connector task by using the **Append Connector** context menu. Follow [our guide on using Connectors](../use-connectors.md) to learn more.
+
+## Make your GraphQL Connector executable
+
+To make the **GraphQL Connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![graphql connector red properties](../img/connectors-graphql-red-properties.png)
+
+:::note
+All the mandatory and non-mandatory fields will be covered in the upcoming sections. Depending on the authentication selection you make, more fields might be required. We will also cover this in the next section.
+:::
+
+### Authentication
+
+You can choose among the available authentication types according to your authentication requirements using the Authentication section.
+
+### None
+
+Click None in the Authentication section. No extra authentication configuration is required; you can jump to the next section.
+
+### Basic
+
+##### Create a new Connector secret
+
+We advise you to keep your **Password** safe and avoid exposing it in the BPMN `xml` file by creating a secret:
+
+1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
+2. Name your secret (i.e `GRAPHQL_BASIC_SECRET`) so you can reference it later in the Connector.
+
+### Configure Basic Authentication
+
+Select the **GraphQL Connector** and fill out the following properties under the **Authentication** section:
+
+1. Click **Basic** in the **Authentication** section.
+2. Set **Username** (i.e. `secrets.YOUR_USERNAME`).
+3. Set **Password** to the secret you created (i.e. `secrets.GRAPHQL_BASIC_SECRET`).
+
+![graphql Connector basic](../img/connectors-graphql-basic.png)
+
+### Bearer Token
+
+#### Create a new Connector secret
+
+We advise you to keep your **Bearer Token** safe and avoid exposing it in the BPMN `xml` file by creating a secret:
+
+1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
+2. Name your secret (i.e `GRAPHQL_BEARER_TOKEN`) so you can reference it later in the Connector.
```

Comment:
I think this is out of sync with the screenshot. The screenshot has `REST` in it